solr/modules/monitor/build.gradle

```
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: 'java-library'
+
+description = 'Apache Solr Monitor'
+
+dependencies {
+
+    implementation project(":solr:core")
+    implementation project(":solr:solrj")
+    implementation "org.apache.lucene:lucene-core"
+    implementation "org.apache.lucene:lucene-monitor"
+    implementation 'com.github.ben-manes.caffeine:caffeine'
```

the CI says

```
Execution failed for task ':solr:modules:monitor:analyzeClassesDependencies'.
> Dependency analysis found issues.
  usedUndeclaredArtifacts
   - io.dropwizard.metrics:metrics-core:4.2.26@jar
```

and a remedy might be (something like)

```suggestion
    implementation 'com.github.ben-manes.caffeine:caffeine'
    implementation 'io.dropwizard.metrics:metrics-core'
```

however with that and `./gradlew clean ; ./gradlew :solr:modules:monitor:analyzeClassesDependencies` locally it then instead says

```
Execution failed for task ':solr:modules:monitor:analyzeClassesDependencies'.
> Dependency analysis found issues.
  unusedDeclaredArtifacts
   - io.dropwizard.metrics:metrics-core:4.2.25@jar
```

instead i.e. the opposite.